### ISSUE TITLE
General Data Protection Regulation (GDPR)

### ISSUE THREAD

**Issue by nelsonic** [dwyl/learn-security#25]
A bit of _light_ evening reading:
https://ico.org.uk/for-organisations/data-protection-reform/overview-of-the-gdpr
🙄 

PDF: https://ico.org.uk/media/for-organisations/data-protection-reform/overview-of-the-gdpr-1-12.pdf

**Comment by iteles** [comment]
We just got this from Google:
<img width="720" alt="screen shot 2017-10-14 at 17 01 46" src="https://user-images.githubusercontent.com/4185328/31577157-7d19cd32-b101-11e7-9702-54bb8121bf7d.png">

**Comment by nelsonic** [comment]
@iteles I'm keeping this issue _open_ because at present 
https://github.com/dwyl/learn-security/tree/2fb68ed37f29f63891f34ffca5be465fc414c928#gdpr
Is the "**What?**" _without_ the "***How***?"

**Comment by iteles** [comment]
Agreed, but I would note that the 'How?' can vary quite a bit from application to application.

**Comment by nelsonic** [comment]
https://www.smashingmagazine.com/2018/02/gdpr-for-web-developers

**Comment by nelsonic** [comment]
![image](https://user-images.githubusercontent.com/194400/41040219-ad788cde-6993-11e8-886b-55ebe85f89f7.png)

That's _one_ way of "_solving_" the issue.
